packages/hash/integration/src/mockData/index.ts

```diff
@@ -85,14 +86,25 @@ void (async () => {
     string
   > = {} as any;
 
+  const componentIdSchema = async (componentId: string) => {
+    const blockSchema = await (
+      await fetch(`${componentId}/block-schema.json`)
+    ).json();
+
+    return { componentId, ...(blockSchema as object) };
+  };
+
   await Promise.all(
     requiredBlockTypes.map(async (name) => {
       const entityType = await EntityType.create(db, {
         accountId: systemOrg.accountId,
         createdByAccountId: systemOrg.entityId, // TODO
         name,
         schema: {
-          componentId: `https://block.blockprotocol.org/${name.toLowerCase()}`,
+          // @todo The componentId given here is not present in the blockPaths.json mapping, blockPaths.json is not needed anymore.
+          ...(await componentIdSchema(
+            `https://blockprotocol.org/blocks/@hash/${name.toLowerCase()}`,
```

Comment:
Our hub may contain blocks in other workspace, e.g. https://github.com/blockprotocol/blockprotocol/pull/183

If we only plan to support `@hash` one for now, it’d be nice to add a todo comment (ideally linking to some task)

Reply:
let's make it support all namespaces from the outside.

We'll shortly want to introduce the ability to search the Block Hub for other blocks to use

Reply:
https://github.com/hashintel/hash/blob/3338697b0c74d4bb676f6c722c4316c01f10ef09/packages/hash/integration/src/mockData/index.ts#L72-L80 could then be a list of lists of `[entityTypeName, componentId]` so that we can support any componentId instead of assuming the location